diffrax/_solver/srk.py

```diff
@@ -0,0 +1,624 @@
+from dataclasses import dataclass
+from typing import ClassVar, Optional
+from typing_extensions import TypeAlias
+
+import equinox as eqx
+import jax.lax as lax
+import jax.numpy as jnp
+import jax.tree_util as jtu
+import numpy as np
+from equinox import AbstractClassVar
```

Comment:
Check the docs on this -- to work around a pyright bug this needs to be imported as
```python
if TYPE_CHECKING:
    from typing import ClassVar as AbstractClassVar
else:
    from equinox import AbstractClassVar
```

Reply:
Hmm, I didn't get any errors, but yes, I will make the edit.